contracts/extensions/votingReputation/VotingReputation.sol

```diff
@@ -94,11 +94,15 @@ contract VotingReputation is ColonyExtension, PatriciaTreeProofs, BasicMetaTrans
   mapping (bytes32 => uint256) expenditureMotionCounts; // expenditure struct signature => count
 
   mapping(address => uint256) metatransactionNonces;
-  function getMetatransactionNonce(address userAddress) override public view returns (uint256 nonce){
+
+  /// @notice Gets the next nonce for a meta-transaction
```

Comment:
I would say that the comments could be removed from this file since we've got an interface contract, but I'll leave that up to you.